docs-developer/local_preview.md

```diff
@@ -0,0 +1,85 @@
+# Getting a local preview of Studio
+
+One of Kolibri Studio's deployment methods use Kubernetes and Docker. If you want a
+quick and easy way to get Studio up and running with minimal configuration,
+this is the method you want to use. This works best on a Linux or MacOS machine.
```

Comment:
Given the discussions we've had about the Docker / kubernetes setup, and the fact that most people switch back to manual, we should probably not market this as the "quick and easy way to get Studio up and running", as I think that's more what we would want it to be than what it actually is right now. ;-) Maybe just remove that sentence entirely, to not recommend nor discourage its use.

Also, why do we call this a "local preview"? Maybe we should just call this "Docker + Kubernetes Studio Instance Setup"?